src/lib/rest/rest.cpp

```diff
@@ -1041,6 +1042,16 @@ static int connectionTreat
       ciP->httpHeaders.servicePath = defaultServicePath(url, method);
     }
 
+    /* X-Forwared-For (used by a potential proxy on top of Orion) overrides ip */
+    if (ciP->httpHeaders.xforwardedFor == "")
+    {
+      LM_TRANSACTION_SET_FROM(ip);
```

Comment:
Ok. However, I would a like bit further and then rename all LM_TRANSACTION_* old-macro-new-inline-functions with names following the usual fucntion convention.